pkg/apis/app/v1/kieapp_types.go

```diff
@@ -74,22 +79,49 @@ type KieAppObject struct {
 }
 
 type Environment struct {
-	Console CustomObject   `json:"console,omitempty"`
-	Others  []CustomObject `json:"others,omitempty"`
-	Servers []CustomObject `json:"servers,omitempty"`
+	Console     CustomObject   `json:"console,omitempty"`
+	Smartrouter CustomObject   `json:"smartrouter,omitempty"`
+	Servers     []CustomObject `json:"servers,omitempty"`
```

Comment:
I haven't fully thought this through yet, but what does it mean for omit to be on an individual CustomObject, whereas there is an array of them?

Reply:
That'd imply to add a new object to have omit + customObject and another one for omit + []customObject. I didn't want to complicate much the current data structure.
```
env.Servers.Omit // bool
env.Servers.Objects[i] // CustomObject
env.Console.Omit // bool
env.Console.Object // CustomObject
```